src/datatypes/array.c

```diff
@@ -50,6 +50,18 @@ bool SIArray_ContainsType(SIValue siarray, SIType t) {
 	return false;
 }
 
+bool SIArray_ContainsValue(SIValue siarray, SIValue value) {
```

Comment:
Please duplicate comments from header file to C file

Reply:
Ok. Done in [aa6194f](https://github.com/RedisGraph/RedisGraph/pull/2807/commits/aa6194fbec956a35195dd72b6149b5636e2208a7)